src/api/app/components/bs_request_comment_component.html.haml

```diff
@@ -17,6 +17,10 @@
             = link_to("#edit_form_of_#{comment.id}", id: "edit_button_of_#{comment.id}", 'data-bs-toggle': 'collapse',
                       class: 'dropdown-item collapsed') do
               Edit
+          - if Flipper.enabled?(:content_moderation, User.session)
+            - if policy(comment).moderate?
+              = button_to(comment.moderated? ? 'Unmoderate' : 'Moderate', comment_moderate_path(comment),
```

Comment:
```suggestion
              = button_to(comment.moderated? ? 'Release' : 'Moderate', comment_moderate_path(comment),
```

Reply:
Relase can be confusing on our context. What about synonyms like: exempt, write off, reprieve... ?